docs/data/date-pickers/base-concepts/base-concepts.md

```diff
@@ -174,3 +174,17 @@ const cleanText = (string) =>
 // Example of a test using the helper
 expect(cleanText(input.value)).to.equal('04-17-2022');
 ```
+
+## Overriding slots and slot props
+
+Date and Time Pickers are complex components built using many subcomponents, known as **slots**. Slots are commonly filled by React components and you can easily override them using the `slots` prop. You can also pass additional props to the available `slots`, using the `slotProps` prop. You can explore a more in-depth walkthrough of the mental model of **slots** on [this page](/base-ui/guides/overriding-component-structure/).
+
+You can find the list of available `slots` for each component in its respective [API reference](/x/api/date-pickers/date-picker/#slots) doc.
+
+Some parts of the pickers' UI are built on several nested slots. For instance, the adornment of the `TextField` on `DatePicker` contains three slots (`inputAdornment`, `openPickerButton`, and `openPickerIcon`) that you can use depending on what you are trying to customize.
```

Comment:
- capitalize Pickers since we're referring to the product, not the general concept

```suggestion
Some parts of the Pickers' UI are built on several nested slots. For instance, the adornment of the `TextField` on `DatePicker` contains three slots (`inputAdornment`, `openPickerButton`, and `openPickerIcon`) that you can use depending on what you are trying to customize.
```